include/zephyr/drivers/video-controls.h

```diff
@@ -134,14 +134,19 @@ enum video_colorfx {
 	VIDEO_COLORFX_ANTIQUE = 14,
 };
 
-/* Enable Automatic Brightness. */
+/** Enable Automatic Brightness. */
 #define VIDEO_CID_AUTOBRIGHTNESS (VIDEO_CID_BASE + 32)
 
 /** Switch the band-stop filter of a camera sensor on or off, or specify its strength.
  * Such band-stop filters can be used, for example, to filter out the fluorescent light component.
  */
 #define VIDEO_CID_BAND_STOP_FILTER (VIDEO_CID_BASE + 33)
 
+/** Rotate the image by a given angle, e.g. 90, 180, 270 degree. The application will be then
+ * responsible for setting the new width and height of the image using video_set_fmt() if needed.
```

Comment:
Drivers often call the implementation with compact name like `_set_fmt()` but applications would likely use [`video_set_format()`](https://docs.zephyrproject.org/latest/doxygen/html/group__video__interface.html#gab93c2cb09bf5b0629b665cc4a079e3cd).